oktoberfest/preprocessing/preprocessing.py

```diff
@@ -159,7 +163,9 @@ def filter_peptides_for_model(peptides: pd.DataFrame, model: str) -> pd.DataFram
     return filter_peptides(peptides, **filter_kwargs)
 
 
-def filter_peptides(peptides: pd.DataFrame, min_length: int, max_length: int, max_charge: int) -> pd.DataFrame:
+def filter_peptides(
+    peptides: Optional[Union[pd.DataFrame, AnnData]], min_length: int, max_length: int, max_charge: int
+) -> Optional[Union[pd.DataFrame, AnnData]]:
```

Comment:
return type is never None! Also I think it should always be a dataframe that is returned(?)

Reply:
removed Optional return and input